### ISSUE TITLE
Empty security array is rendered into all operations

### ISSUE THREAD

**Issue by derrabus** [nelmio/NelmioApiDocBundle#1695]
### Problem

I'm using NelmioApiDocBundle 4.0.0-beta1 in a Symfony 4.4 application. My API is using the same authentication method for all endpoints, so I've configured authentication globally for the whole API as described in https://swagger.io/docs/specification/authentication/

```yaml
nelmio_api_doc:
    documentation:
        components:
            securitySchemes:
                SymfonySession:
                    type: apiKey
                    in: cookie
                    name: '%app.session_cookie_name%'
        security:
            - SymfonySession: []
```

This should do the trick. However, in the generated output, all operations receive an empty `security` array:

```JSON
"\/my\/endpoint": {
    "get": {
        "security":[]
    }
}
```

This defeats my global setting which is very unfortunate. I could add an annotation `@Security(name="SymfonySession")` to all of my endpoints. However, this method adds a lot of boilerplate and is error-prone since that annotation is easily forgotten when adding a new controller to the API.

### Suggestion
* Don't render `security` inside an operation unless security has explicitly been configured for that operation.
* Offer an easy way to force `security: []` on a specific operation.

**Comment by GuilhemN** [comment]
Thanks for the report! Your issue should be fixed by https://github.com/nelmio/NelmioApiDocBundle/pull/1700.

It seems we were misusing `operation->security` in the `RouteMetadataDescriber`, it was putting the route schemes constraint in `security` although this field isn't suited.
I simply removed this part of the code, it seems there is no way to enforce a scheme cleanly from the bundle in OpenApi 3 (there is operation->servers but it needs a full URL, and setting it would cause more harm than just letting the user deal with this setting imo).

Ps: love your work on symfony by the way :smile:

**Comment by derrabus** [comment]
> Thanks for the report! Your issue should be fixed by #1700.

Thanks a lot for the quick fix! I will try out your change and give feedback.

> Ps: by the way I love your work on symfony 😄

Thank you, that's very kind. :-)

**Comment by derrabus** [comment]
Sorry, it took me so long. I can confirm, that the problem is solved on master. Thanks a lot. :-)
